posthog/hogql/constants.py

```diff
@@ -67,7 +68,8 @@
     # array functions
     "tuple": "tuple",
     # conditional
-    "ifElse": "if",
+    "if": "if",
```

Comment:
This converts our `if` function back to the clickhouse standard `if`. Previously we used `ifElse` because `if` is a reserved keyword in python and couldn't be used as a call name.